docs/docs/Support/troubleshooting.mdx

```diff
@@ -124,6 +124,55 @@ There are two possible reasons for this error:
 Environment variables set in your terminal aren't automatically available to GUI-based applications like Langflow Desktop when launched through the Finder or the Start Menu.
 To set environment variables for Langflow Desktop, see [Set environment variables for Langflow Desktop](/environment-variables#set-environment-variables-for-langflow-desktop).
 
+### Access Langflow Desktop startup logs
```

Comment:
Do you think this should be on the logging page?

If not, we should add a link on the logging page to this section. It could be under [See also](https://docs.langflow.org/logging#see-also).

Reply:
Added to logging and linked it from troubleshooting